### DISCUSSION TITLE
posts/678

### DISCUSSION THREAD

**Discussion by giscus** [kakaotech/comments#36] [Announcements]
# posts/678

실시간 메시징 시스템 개발 시리즈

👉 Part 1. 실시간 메시징 시스템 개...

https://tech.kakao.com/posts/678?utm_source=oneoneone

**Comment by bearstark** [comment]
글 잘 읽었습니다! 읽던 중에 질문드립니다!!

현재 메인 저장소인 redis의 장애 발생시 시나리오나 보장 범위는 어떻게 잡고 계시나요?
redis의 경우 영속성 설정 중 가장 가성비가 좋은 aof every second sync를 했을 때 최대 1초 단위의 누락이 발생 가능한데 이 점은 감안하고 선택하신 걸까요??
혹시 어떤 영속성 설정을 채택하셨나요?


그리고 redis는 어떤 모드로 운용 중이신가요? cluster이신가요? 아니면 애플리케이션 레벨의 샤딩이신가요??
